bigbluebutton-html5/imports/ui/services/audio-manager/index.js

```diff
@@ -717,14 +717,18 @@ class AudioManager {
   }
 
   onVoiceUserChanges(fields = {}) {
-    if (fields.muted !== undefined && fields.muted !== this.isMuted) {
+    // when user leaves voice conf, set muted = false
+    // as the user might have been transfered to a breakout room
+    if (fields.leftVoiceConf !== undefined && fields.leftVoiceConf) {
+      this.isMuted = false;
+    } else if (fields.muted !== undefined && fields.muted !== this.isMuted) {
```

Comment:
This is a legacy workaround from Meteor, where `muted=false` was set when a user left the voice conference.
To avoid keeping the user with the wrong state for everyone, I introduced a new prop `leftVoiceConf` and only the same user will keep `muted=false` and only here.

A review from @prlanzarin would be appreciated to confirm that this change won’t introduce any issues.